ipv8/attestation/trustchain/block.py

```diff
@@ -26,8 +27,73 @@ class TrustChainBlock(object):
     """
     Container for TrustChain block information
     """
+    Data = namedtuple('Data', ['type',
+                               'transaction',
+                               'public_key',
+                               'sequence_number',
+                               'link_public_key',
+                               'link_sequence_number',
+                               'previous_hash',
+                               'signature',
+                               'timestamp',
+                               'insert_time'])
+    """
+    Data struct to initialize a TrustChainBlock.
+
+
+        **[0] type:** The block type name, as utf-8 string or utf-8 encoded bytes.
+
+        *> type:* str or bytes
+
+        **[1] transaction:** Metadata dictionary, consisting of binary type strings.
+
+        *> type:* dict
+
+        **[2] public_key:** The serialized public key of the initiator of this block (binary data).
+
+        *> type:* bytes (Py3) or str (Py2)
+
+        **[3] sequence_number:** The height of this block in the chain of the intiator of this block.
```

Comment:
I wasn't able to immediately understand what "height of the block" means. Maybe change to something like "the serial number of this block"?

Reply:
I would simply say 'the sequence number of this block in the chain of ...'